AsyncDisplayKit/Layout/ASLayoutOptions.mm

```diff
@@ -73,59 +86,57 @@ - (instancetype)initWithLayoutable:(id<ASLayoutable>)layoutable;
 - (id)copyWithZone:(NSZone *)zone
 {
   ASLayoutOptions *copy = [[[self class] alloc] init];
-  [self copyIntoOptions:copy];
+  [copy propagateOptionsFromLayoutOptions:self];
   return copy;
 }
 
-- (void)copyIntoOptions:(ASLayoutOptions *)copy
+- (void)propagateOptionsFromLayoutOptions:(ASLayoutOptions *)layoutOptions
```

Comment:
Yeah, this looks like adopt/overwrite/something.  In fact the original name but flipped might be OK: copyFromOptions: